Dockerfile

```diff
@@ -1,19 +1,25 @@
 # vim:set ft=dockerfile:
 FROM condaforge/mambaforge
-MAINTAINER https://github.com/bird-house/finch
+ARG DEBIAN_FRONTEND=noninteractive
+ENV PIP_ROOT_USER_ACTION=ignore
+LABEL org.opencontainers.image.authors="bourgault.pascal@ouranos.ca"
 LABEL Description="Finch WPS" Vendor="Birdhouse" Version="0.11.1"
 
+# Switch to /code directory
 WORKDIR /code
+COPY . ./
 
+# Build finch environment
 COPY environment.yml .
-RUN mamba env create -n finch -f environment.yml \
-    && mamba install -n finch gunicorn psycopg2 \
-    && mamba clean -ay
-
-COPY . .
+RUN mamba env create -n finch -f environment.yml
+RUN mamba install -n finch gunicorn psycopg2
+RUN mamba clean --all --yes
 
+# Add the finch conda environment to the path
 ENV PATH /opt/conda/envs/finch/bin:$PATH
 
-EXPOSE 5000
+RUN pip install . --no-deps
```

Comment:
Oh interesting.  I never notice the was no pip install for Finch !!!  This is harmless but does it brings anything more other than adding another layer and consuming a bit more spaces?

Reply:
I don't believe it adds much to the resources (I specified --no-deps to ensure we don't pull PyPI deps). It does offer the ability to `RUN` the service using `$ finch`, so that's nice.

Reply:
right ok, keep this